test/libsolidity/ASTJSON/documentation_triple.sol

```diff
@@ -0,0 +1,15 @@
+contract C {
+    /// test
+    uint a;
+    function f() public pure returns (uint x) {
+        /// test2
+        for (uint i = 0; i < 20; i++) {
+            /// test3
+            x *= 2;
+        }
+        /// test4
```

Comment:
Can you add a natspec comment with spaces and quotation marks to ensure it is stored properly (escaped if needed)?